### ISSUE TITLE
There are a few issues with running MAKE_pairs

### ISSUE THREAD

**Issue by hanyunlong233** [HeatherJiaZG/SuperGlue-pytorch#5]
In superglue model, the input date is a dictionary which contains the key 'all_matches' and 'file_name '.So when train ,the SparseDataset produced training data containing these keys.But the evalution input did not contain any of this, so an error was reported.The 'file_name' is not used in the model and can be added directly, but 'all_matches' is used in the model and I hope you can give me a modified code or give me some comments.
Thank You!

**Comment by sunset326** [comment]
do you solve this problem ,brother.  when i evaluate it, i also notice this problem you propose

**Comment by Albert337** [comment]
yeah,when i checked the load_data.py , i found that the all_matches were not meant for the only matcher pair of two images, was that reasonable?
